Periodic_3_mesh_3/doc/Periodic_3_mesh_3/Periodic_3_mesh_3.txt

```diff
@@ -0,0 +1,705 @@
+
+\subsection Periodic_3_mesh_3TheGlobalFunctions The Global Functions
+
+A periodic 3D mesh generation process is launched through a call
+to one of the two following functions:
+
+\code{.cpp}
+template <class C3T3, class PeriodicMeshDomain, class MeshCriteria>
+C3T3 make_periodic_3_mesh_3(MeshDomain domain,
+                            MeshCriteria criteria,
+                            parameters::internal::Features_options features = parameters::features(domain),
+                            parameters::internal::Lloyd_options lloyd = parameters::no_lloyd(),
+                            parameters::internal::Odt_options odt = parameters::no_odt(),
+                            parameters::internal::Perturb_options perturb = parameters::perturb(),
+                            parameters::internal::Exude_options exude = parameters::exude());
+
+template <class C3T3, class MeshDomain, class MeshCriteria>
+void refine_periodic_3_mesh_3(C3T3& c3t3,
+                              PeriodicMeshDomain domain,
+                              MeshCriteria criteria,
+                              parameters::internal::Lloyd_options lloyd = parameters::no_lloyd(),
+                              parameters::internal::Odt_options odt = parameters::no_odt(),
+                              parameters::internal::Perturb_options perturb = parameters::perturb(),
+                              parameters::internal::Exude_options exude = parameters::exude());
+\endcode
+
+The function `make_periodic_3_mesh_3()` generates from scratch a periodic mesh
+of the input domain, while the function `refine_periodic_3_mesh_3()` refines
+an existing periodic mesh of the input domain.
+
+\warning The triangulation must form at all times a simplicial complex within
+a single copy of the domain (see Sections \ref P3Triangulation3secspace and \ref P3Triangulation3secintro
+of the manual of 3D periodic triangulations). It is the responsibility of the user to provide
+a triangulation that satisfies this condition when calling the refinement
+function `refine_periodic_3_mesh_3()`. The underlying triangulation of a mesh
+complex obtained through `make_periodic_3_mesh_3()` or `refine_periodic_3_mesh_3()`
+will always satisfy this condition.
+
+\subsection Periodic_3_mesh_3TheDataStructure The Data Structure
+
+The template parameter `C3T3` is required to be a model of
+the concept `MeshComplex_3InTriangulation_3`. This data structure is devised to
+represent a three-dimensional complex embedded in a periodic 3D triangulation.
+In both functions, an instance of type `C3T3` is used to maintain the current
+approximating simplicial mesh and to represent a single copy of
+the final periodic 3D mesh at the end of the procedure.
+
+The embedding periodic 3D triangulation is required to be the nested type
+`CGAL::Periodic_3_mesh_triangulation_3::type`, provided by the class
+`CGAL::Periodic_3_mesh_triangulation_3`. The type for this triangulation is a
```

Comment:
`CGAL::Periodic_3_mesh_triangulation_3` is not a class. That is a class template. You can also call it a meta-functor, but that wording would need a link, because it is not well known.